include/array/vector.h

```diff
@@ -121,29 +124,28 @@ int32_t VectorInsert(Vector *self, Item item, int32_t iIdx);
 /**
  * @brief Pop an item from the tail of the vector.
  *
- * This function removes an item from the tail of the vector. If the knob is on,
- * it also runs the resource clean method for the removed item. Naturally, the
- * vector size is shrunk.
+ * This function removes an item from the tail of the vector. If user defined
+ * destroy func is set, it also runs the resource clean method for the removed 
+ * item. Naturally, the vector size is shrunk.
  *
  * @param self          The pointer to the Vector structure
- * @param bClean        The knob to clean item resource
  *
  * @retval SUCC
  * @retval ERR_NOINIT   Uninitialized container
  * @retval ERR_IDX      Empty vector
  */
-int32_t VectorPopBack(Vector *self, bool bClean);
+int32_t VectorPopBack(Vector *self);
 
 /**
  * @brief Delete an item from the designated index of the vector.
  *
  * This function removes an item from the designated index of the vector and shifts
- * the trailing items one position to the head. If the knob is on, it also runs the
- * resource clean method for the removed item. Naturally, the vector size is shrunk.
+ * the trailing items one position to the head. If user defined destroy func is
+ * set, it also runs the resource clean method for the removed item. Naturally, 
+ * the vector size is shrunk.
```

Comment:
We may keep the formal syntax for the comment block.
Thus for "If user defined destroy func is set", the following sentence would be better:
"If the custom resource clean method is set, it also runs the clean method for the removed item."